benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadServer.java

```diff
@@ -141,7 +142,7 @@
     if (config.getServerType() == Control.ServerType.ASYNC_GENERIC_SERVER) {
       serverBuilder.addService(
           ServerServiceDefinition
-              .builder(BenchmarkServiceGrpc.SERVICE_NAME)
+              .builder(new ServiceDescriptor(BenchmarkServiceGrpc.SERVICE_NAME))
```

Comment:
It looks like this is missing `GENERIC_STREAMING_PING_PONG_METHOD`. Should ServerServiceDefinition verify consistency with the ServiceDescriptor?